README.md

```diff
@@ -58,8 +58,11 @@ installation and postprocessing procedures. Its features include:
 * Discontinuous Galerkin methods
   * Kinetic energy-preserving and entropy-stable methods based on flux differencing
   * Entropy-stable shock capturing
-  * Positivity-preserving limiting
   * [Finite difference summation by parts (SBP) methods](https://github.com/ranocha/SummationByPartsOperators.jl)
+* Advanced limiting strategies
+  * Positivity-preserving limiting
```

Comment:
Thanks for adding the IDP limiting.
The positivity-preserving limiting just includes the limiter of zhang, shu here, right? Maybe we should add something like `(global (positivity-preserving) and local bounds (IDP shock-capturing))` to the IDP limiter? But I'm not sure what's best.

Reply:
Does the IDP stuff also only boil down to positivity preservation, or is there some suppression of oscillations involved?

Reply:
I also noticed that the SSPRK33 in Trixi is also still marked as experimental:

https://github.com/trixi-framework/Trixi.jl/blob/288d41d61fa357df6fa10dbfe1db514f6be6a59b/src/time_integration/methods_SSP.jl#L23-L26

Question whether we would want to put this feature than so prominent on the landing page.

Same holds for 

https://github.com/trixi-framework/Trixi.jl/blob/288d41d61fa357df6fa10dbfe1db514f6be6a59b/src/solvers/dgsem_tree/subcell_limiters.jl#L64-L67

https://github.com/trixi-framework/Trixi.jl/blob/288d41d61fa357df6fa10dbfe1db514f6be6a59b/src/callbacks_stage/subcell_limiter_idp_correction.jl#L27-L30

Reply:
> Does the IDP stuff also only boil down to positivity preservation

Reply:
Oh yeah right. Although, I think it is time to remove those remarks from the documentary. I will add this point to the Trixi agenda and open a PR then.
Since @sloede will not be there tomorrow, I ping him here and ask for his opinion.